django_components/template_loader.py

```diff
@@ -0,0 +1,12 @@
+"""
+Wrapper for loading templates from "components" directories in INSTALLED_APPS
+packages.
```

Comment:
Suggestion to change this to: Template loader that loads templates from each Django app's "components" directory.

Reply:
That's just a copy paste from https://github.com/django/django/blob/main/django/template/loaders/app_directories.py :D